src/main/drivers/serial_uart.c

```diff
@@ -51,18 +52,23 @@ static void usartConfigurePinInversion(uartPort_t *uartPort) {
 #ifdef STM32F303xC
     uint32_t inversionPins = 0;
 
-    if (uartPort->port.mode & MODE_TX) {
-        inversionPins |= USART_InvPin_Tx;
-    }
-    if (uartPort->port.mode & MODE_RX) {
-        inversionPins |= USART_InvPin_Rx;
+    // Inversion when using OPTION_BIDIR not supported yet.
+    if (uartPort->port.options & SERIAL_BIDIR) {
```

Comment:
Don't know, I was just following the functionality of the old code (which didn't enable inverters for BIDIR since MODE_TX and MODE_RX were not set).